docs/docs/troubleshooting-common-errors.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Troubleshooting Common Errors
+---
+
+As you encounter errors while developing with Gatsby, it is likely you'll run into something that other users have already stumbled upon. Some errors may require fixes to Gatsby's core processes, these are best filed as [issues](/contributing/how-to-file-an-issue/). Many errors you encounter will mean adjusting how you've configured a plugin or API in your site.
+
+This guide is meant as a reference for common errors that have tripped up other Gatsby users.
+
+## Problems with the cache
+
+Running a site in `gatsby develop` will set up a server locally that enables features like [hot-module replacement](/glossary#hot-module-replacement). Gatsby keeps a cache of data and rendered assets in the `.cache` folder at the root of a Gatsby site so that it doesn't have to repeat work processing optimized resources. If you see errors about not being able to find a resource in the cache it may be enough to clear your cache and restart your server. You can clear Gatsby's cache by running;
```

Comment:
Fixes a broken link and replaces a semi-colon with a colon
```suggestion
Running a site in `gatsby develop` will set up a server locally that enables features like [hot-module replacement](/docs/glossary#hot-module-replacement). Gatsby keeps a cache of data and rendered assets in the `.cache` folder at the root of a Gatsby site so that it doesn't have to repeat work processing optimized resources. If you see errors about not being able to find a resource in the cache it may be enough to clear your cache and restart your server. You can clear Gatsby's cache by running:
```